code/GameEngine/Components/Effects/ParticleSystem.cs

```diff
@@ -8,12 +8,31 @@
 public class ParticleSystem : BaseComponent, BaseComponent.ExecuteInEditor
 {
 	Sandbox.ParticleSystem _particles;
-
+	
 	[Property] public bool Looped { get; set; } = false;
 
 	[Range( 0, 2.0f )]
 	[Property] public float PlaybackSpeed { get; set; } = 1.0f;
 
+	private bool emissionStopped;
+	
+	/// <summary>
+	/// Turn on or off particle emission.
+	/// Useful for particles with intermittent or permanent durations.
+	/// </summary>
+	[Property]
+	public bool EmissionStopped
```

Comment:
We should flip this so it's just "Emission", default to true

Reply:
Updated it 👍 

I ran into some situations where setting Emission back to true after disabling it didn't re-enable the particle effect. It seems at some point it's considered finished if the emission is stopped and all the active particles decay. 

Not necessarily a problem but it makes me wonder if changing Emission to true should recreate the scene object.